scripts/setup.sh

```diff
@@ -387,7 +387,7 @@ fi
 
 echo -e
 echo "- schedule the runs"
-if [[ $(gcloud scheduler jobs list --filter=name:$GCP_SCHEDULER_NAME --format="value(name)") ]]; then
+if [[ $(gcloud scheduler jobs list --filter="name ~ $GCP_SCHEDULER_NAME$" --format="value(name)") ]]; then
```

Comment:
`$` should be here?

Reply:
yes, actually it's regex pattern, the format with ":" is deprecated for scheduler jobs list command

Reply:
```
WARNING: --filter : operator evaluation is changing for consistency across Google APIs. 
name:dynatrace-gcp-schedule currently matches but will not match in the near future.
```